stock_picking_report_by_partner/README.rst

```diff
@@ -0,0 +1,39 @@
+Picking Report by Partner
+=========================
+
+This module was written to extend the functionality of picking reports to support diferent picking report by partner and allow you to set the picking report template in partner form to print this template as default.
+
+Usage
+=====
+
+To use this module, you need to:
+
+ * go to partner configuration
+ * set default picking report template
+ * print picking report
+
```

Comment:
Could you please add 'Try me on Runbot' button and 'Bug Tracker' section?
See https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst as reference.